handbook/engineering/campaigns/index.md

```diff
@@ -45,30 +45,46 @@ When I create a campaign to make large-scale code changes I want to _focus on th
 
 ## Process
 
-For each iteration (currently one month long), we follow this process:
+* We prioritize our work in [this project](https://github.com/sourcegraph/sourcegraph/projects/10). It is the responsibility of the PM and EM to keep this updated and correct.
+
+* Each day, Slack reminds us to do our text check-in, which consists of a *short* message (it shouldn't take longer than a minute to write) in the reminder's thread. This should be a recap of what we have finished that day.
+
+* On Tuesdays, each engineer posts a status update in the current tracking issue with any additional info regarding what they accomplished in the previous week, as well as what they intend to accomplish in the coming week. The EM then rolls this up into a status update that is emailed to engineering leadership.
```

Comment:
Since this is new, we might need to clarify this:

```suggestion
* On Tuesdays, each engineer posts a status update in the current tracking issue with any additional info regarding what they accomplished in the previous week that wasn't captured in their daily status updates, as well as what they intend to accomplish in the coming week. The EM then rolls this up into a status update that is emailed to engineering leadership.
```